src/scrabble.js

```diff
@@ -1,5 +1,81 @@
-function scrabble() {
-  // write code here
+const { letters } = require('./letter.js')
+
+const scrabble = (word) => {
+  let output = 0
+  let mulitplier = 1
+  // Arrays to test for valid input
+  const checkerArr = []
+  const checkerArr2 = []
+
+  // Checking for any number in input
+  function checkString(string) {
+    return /^[0-9]*$/.test(string)
```

Comment:
I don't understand the use of   /^[0-9]*$/

Reply:
It's RegEx. Checking if there's any numbers (0-9) in the string given, on line 15.